internal/exec/util/file.go

```diff
@@ -112,6 +114,33 @@ func RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
 		return nil
 	}
 
+	if f.User.Name != "" {
+		u, err := user.Lookup(f.User.Name)
```

Comment:
Oh, I assumed you were talking about the one in `os/user`, I completely missed this one. Yeah, I'll switch it to use that. And come up with a more practical test.